app/assets/stylesheets/errors.scss

```diff
@@ -0,0 +1,97 @@
+@import 'colors';
+
+.error-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.error-svg {
+  margin-right: 30px;
+}
+
+.error-title {
+  color: $black-color;
+}
+
+.error-text {
+  color: $black-color;
+  font-size: 27px;
+}
+
+.message-box {
+  width: 380px;
+  font-weight: 300;
+}
+
+.message-box h1 {
+  font-size: 100px;
+  line-height: 46px;
+  margin-bottom: 40px;
+}
+
+.error-button .action-link-wrap {
+  margin-top: 40px;
+
+  a {
+    background: $black-color;
+    padding: 8px 25px;
+    border-radius: 4px;
+    color: $white-color;
+    font-weight: 700;
+    font-size: 14px;
+    cursor: pointer;
+    text-decoration: none;
+    margin-right: 10px;
+
+    &:hover {
+      background: $blue-color;
+      color: $white-color;
+    }
+  }
+}
+
+#Polygon-1 , #Polygon-2 , #Polygon-3 , #Polygon-4, #Polygon-5 {
```

Comment:
for future:
#Polygon-1 , 
#Polygon-2 , 
#Polygon-3 , 
#Polygon-4, 
#Polygon-5 {
...
}

Reply:
ok, thanks